src/test/java/tests/bankManager/BankManagerAccountTests.java

```diff
@@ -1,25 +1,47 @@
 package tests.bankManager;
 
 import baseClasses.BankManagerLoginPage;
+import baseClasses.CustomerLoginPage;
 import baseClasses.HomePage;
 import baseTest.AbstractBaseTest;
+import com.codeborne.selenide.Condition;
+import org.testng.Assert;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
 public class BankManagerAccountTests extends AbstractBaseTest {
 
     private HomePage homePage;
     private BankManagerLoginPage bankManagerLoginPage;
+    private CustomerLoginPage customerLoginPage;
     private Assertions assertions;
 
 
     @BeforeMethod
     public void openBankHomePage() {
         homePage = new HomePage();
         bankManagerLoginPage = new BankManagerLoginPage();
+        customerLoginPage = new CustomerLoginPage();
         assertions = new Assertions();
         homePage.open();
     }
+    public void regNewCustomerAccount(String firstName, String lastName, String postCode, String nameOfSelectedToEnter, String currency){
```

Comment:
a lot of arguments. 
https://habr.com/ru/companies/otus/articles/552412/
use pattern biulder